modules/rest-api/pages/rest-xdcr-create-replication.adoc

```diff
@@ -101,6 +102,19 @@ A value of `true` means that the replication should be allowed to continue witho
 A value of `false` (the default) means that the replication should be restarted.
 For more information, see xref:learn:clusters-and-availability/xdcr-filtering.adoc#filter-expression-editing[Filter-Expression Editing].
 
+The `filterBinary` flag is provided only in Couchbase Server Version 7.1.5+, and specifies whether binary documents should be replicated.
+The value can be `true` or `false` (the default).
+If the value is `true`, binary documents are _not_ replicated, regardless of whether a `filterExpression` is applied.
+If the value is `false`:
+
+* If a filter expression is _not_ provided, binary documents _are_ replicated.
+
+* If a filter expression is provided, and the expression refers only to either the document's _key_, or its _xattr_, or to both, the expression is applied, and the document is replicated if the expression permits.
+
+* If a filter expression is provided, and the expression refers to the document's body, the document is _not_ replicated.
+
+* If a filter expression is provided, and the expression refers to the document's _key_, or its _xattr_, or to both; and also refers to the document's body; then the expression is applied to the _key_ and/or _xattr_, and the document is replicated if the expression permits.
```

Comment:
* If a filter expression is provided, and the expression refers to the document's _key_, or its _xattr_, or to both; and also refers to the document's body; then the document is not replicated -- regardless of whether or not the _key_ or the _xattr_ expression permits.

Reply:
I realize that there is no logic to the behavior when filterBinary is false and the scenarios involving when an expression refers to the document's body -- it's just the behavior.  It's the same behavior when filterBinary did not exist -- so, the behavior when filterBinary is false is the same behavior when there was no filterBinary in how binary documents were treated during replication in various scenarios.   So, if you just want the same behavior you always had in previous versions, set filterBinary to false.

The documentation should note that the behavior of filterBinary when it is false is the behavior of XDCR in regards to binary documents in versions where filterBinary option did not exist.